src/backend/dx11/src/device.rs

```diff
@@ -1225,12 +1385,26 @@ impl hal::Device<Backend> for Device {
             view_kind,
             format: conv::map_format(format)
                 .ok_or(image::ViewError::BadFormat)?,
+            range: range.clone(),
+        };
+
+        let srv_info = ViewInfo {
+            resource: image.internal.raw,
+            kind: image.kind,
+            flags: image.storage_flags,
+            view_kind,
+            format: match info.format {
```

Comment:
this match (a similar one, at least) is also repeated quite a few times, let's share it